src/login/70-uaccess.rules.in

```diff
@@ -80,4 +80,7 @@ ENV{ID_SOFTWARE_RADIO}=="?*", TAG+="uaccess"
 # 3D printers, CNC machines, laser cutters, 3D scanners, etc.
 ENV{ID_MAKER_TOOL}=="?*", TAG+="uaccess"
 
+# Protocol analysers
+ENV{ID_PROTOCOL_ANALYSER}=="?*", TAG+="uaccess"
```

Comment:
All USB devices have a device node in `/dev/bus/usb...`

I've added the check though.

Reply:
I'be removed the check now, as this should only tag the device node for the device. So it would tag uaccess for the usb_device (`/dev/bus/usb/...`) for the devices I'm adding in this PR, but it could also be used to tag the serial port on a logic analyser if that's the interface that one could use.